### ISSUE TITLE
GH Releases (since this commit)

### ISSUE THREAD

**Issue by itachi1706** [itachi1706/tampermonkey-scripts#2]
Script to check if this exists. If it doesnt, query and inject the value yourself

```javascript
$("<id>").load("<compare URL>", function(data) { $("<id>").text($(".tabnav a:eq(0) .Counter").text().trim()); console.log($(".tabnav a:eq(0) .Counter").text().trim()); });
```

**Comment by itachi1706** [comment]
Probably either call .load() after injecting a div that will contain the information (http://api.jquery.com/load/#loading-page-fragments)

or use jQuery Ajax to handle it and insert element after (http://api.jquery.com/jQuery.get/)

**Comment by itachi1706** [comment]
Find all with this to not get the tagged-only entries
```javascript
$(".release-entry .release.label-latest, .release-entry .release.label-")
```

**Comment by itachi1706** [comment]
Implemented as of 9bff05442386eaf2917a37ea86c6a956ffe5accb
